test/e2e/space_test.go

```diff
@@ -171,7 +172,18 @@ func TestRetargetSpace(t *testing.T) {
 		err := hostAwait.CreateWithCleanup(context.TODO(), space)
 		require.NoError(t, err)
 		// wait until Space has been provisioned on member-1
-		space = VerifyResourcesProvisionedForSpaceWithTier(t, awaitilities, member1Await, space.Name, "base")
+		VerifyResourcesProvisionedForSpaceWithTier(t, awaitilities, member1Await, space.Name, "base")
+
+		// retrieve the latest copy of the Space before updating
+		spaceName := space.Name
+		space = &toolchainv1alpha1.Space{}
+		err = hostAwait.Client.Get(context.TODO(),
+			types.NamespacedName{
+				Namespace: hostAwait.Namespace,
+				Name:      spaceName,
+			},
+			space)
+		require.NoError(t, err)
```

Comment:
Even this code could fail for the conflict and it's impossible to reuse it somewhere else. 
Could you please use the same approach as we use for UserSignup updates? https://github.com/codeready-toolchain/toolchain-e2e/blob/master/testsupport/wait/host.go#L227-L244

Reply:
oops, forgot about that. Thanks for handling it